SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/ChannelGroup.cs

```diff
@@ -99,6 +100,15 @@ public void SetEnabled(bool enabled)
 
         public async Task ShutdownAsync()
         {
+            lock (_channelGroupLock)
```

Comment:
Just out of curiosity, why is this before the call to `ThrowIfDisposed()`?

Reply:
Doesn't matter I guess. Will change, not big deal